src/types.rs

```diff
@@ -449,6 +449,15 @@ pub enum VectorialStrafingConstraints {
         /// The highest yaw angle of the range in degrees.
         to: f32,
     },
+    /// Constrains the player yaw to given point.
+    LookAt {
+        /// Option to trace an entity's origin from entity index.
+        entity: u32,
```

Comment:
```suggestion
        entity: Option<NonZeroU32>,
```
and change the code accordingly